components/BottomAppBar/src/MDCBottomAppBarView.m

```diff
@@ -56,16 +56,14 @@ @interface MDCBottomAppBarView () <CAAnimationDelegate>
 @implementation MDCBottomAppBarView
 
 - (instancetype)initWithFrame:(CGRect)frame {
-  self = [super initWithFrame:frame];
-  if (self) {
+  if (self = [super initWithFrame:frame]) {
```

Comment:
The previous style is [our repo's defacto style](https://github.com/material-components/material-components-ios/search?q=%22if+%28self%29+%7B%22&unscoped_q=%22if+%28self%29+%7B%22), please revert these changes.

Reply:
Sure. Addressed in f4423aa30b94ed3b92a7868fcebfcfdcac58b3d7.